### ISSUE TITLE
Question: plan to re-introduce Auto Metric Observers?

### ISSUE THREAD

**Issue by pmlk** [MetinSeylan/Nestjs-OpenTelemetry#50]
I see that recently with #49 support for Auto Metric Observers was dropped. I'm assuming this is mostly due the dependency on experimental `@opentelemetry` packages which constantly introduce breaking changes.

**Is there a plan to re-introduce these Auto Metrics Observers once the now experimental dependencies are stable?**

**Comment by MetinSeylan** [comment]
yes and under-development without OpenTelemetry metric sdk, it will be released in next version

**Comment by pmlk** [comment]
Awesome! Thank you!
